client/src/pages/OrganizationProfile.js

```diff
@@ -135,30 +139,70 @@ const OrganizationProfile = () => {
     })();
   }, [orgProfileDispatch, organizationId, userProfileDispatch]);
 
+  const fetchOrganizationPosts = async () => {
+    postsDispatch({ type: FETCH_POSTS });
+    try {
+      const res = await axios.get(
+        `/api/posts?limit=-1&authorId=${organizationId}`,
+      );
+      postsDispatch({
+        type: SET_POSTS,
+        posts: res.data,
+      });
+    } catch (err) {
+      const message = err.response?.data?.message || err.message;
+      postsDispatch({
+        type: ERROR_POSTS,
+        error: `Failed loading activity, reason: ${message}`,
+      });
+    }
+  };
+
   useEffect(() => {
-    (async function fetchOrganizationPosts() {
-      postsDispatch({ type: FETCH_POSTS });
-      try {
-        const res = await axios.get(
-          `/api/posts?limit=-1&authorId=${organizationId}`,
-        );
-        postsDispatch({
-          type: SET_POSTS,
-          posts: res.data,
-        });
-      } catch (err) {
-        const message = err.response?.data?.message || err.message;
-        postsDispatch({
-          type: ERROR_POSTS,
-          error: `Failed loading acitivity, reason: ${message}`,
-        });
-      }
-    })();
+    (fetchOrganizationPosts)()
   }, [organizationId]);
```

Comment:
`}, [organizationId]); // eslint-disable-line react-hooks/exhaustive-deps` or it will show a console warning.